### PR TITLE
[Classifier]: Progress bar for validation 

### PR DESCRIPTION
**Pull request by AyushExel**
## 🛠️ PR Summary

<sub>Made with ❤️ by [Ultralytics Actions](https://github.com/ultralytics/actions)<sub>

### 🌟 Summary
Improved progress bar and accuracy reporting in model validation phase.

### 📊 Key Changes
- Added a variable `n` to store the number of batches in the data loader.
- Implemented a tqdm progress bar for the validation step to provide real-time feedback.
- Adjusted the loss calculation to use the new `n` variable for better code clarity.

### 🎯 Purpose & Impact
- **Enhanced User Experience:** The addition of a tqdm progress bar makes it easier for users to track the validation process, leading to a more interactive and informative experience. 📈
- **Improved Code Clarity:** Using the `n` variable for the number of batches simplifies how the loss is calculated and reported, reducing potential confusion for maintainers and contributors. 🧐
- **Accurate Metrics Display:** The progress bar now more accurately displays loss and accuracy, resulting in more reliable performance metrics for users. 🔍

### CODE REVIEW

**Comment by glenn-jocher** [comment]
@AyushExel this one needs some work.

```python
!git clone https://github.com/AyushExel/yolov5 -b tqdm # clone
%cd yolov5
%pip install -qr requirements.txt  # install

import torch
import utils
display = utils.notebook_init()  # checks

!python classifier.py --model yolov5s --data mnist --epochs 5 --img 128
```

Screen Shot 2022-06-29 at 12 46 38 PM

**Comment by AyushExel** [comment]
@glenn-jocher okay looking

**Comment by AyushExel** [comment]
@glenn-jocher this should be fixed now. pbar is being passed from train to test so I've set the test bar to dissappear once the test operation is over. The info will be appended in the main pbar being passed from the train function
Screenshot 2022-06-29 at 10 02 48 PM
.

**Comment by glenn-jocher** [comment]
@AyushExel PR is merged. Thank you for your contributions to YOLOv5 🚀 and Vision AI ⭐
